astroquery/solarsystem/pds/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""
+RingNode
+--------
+
+:author: Ned Molter (emolter@berkeley.edu)
+"""
+
+from astropy import config as _config
+
+
+class Conf(_config.ConfigNamespace):
+    """
+    Configuration parameters for `astroquery.solarsystem.pds`.
+    """
+
+    # server settings
+    pds_server = _config.ConfigItem(
+        ["https://pds-rings.seti.org/cgi-bin/tools/viewer3_xxx.pl?"], "Ring Node"
+    )
```

Comment:
If `ConfigItem` is created using a `list` (like is done here) then that `list` contains all allowed values, unless the `cfgtype='list'` is explicitly used, see https://docs.astropy.org/en/stable/config/index.html#item-types-and-validation for more details.

This configuration item has only one allowed value, which means it is not configurable at all and should not be managed by the configuration system.